OpenRA.Game/ModMetadata.cs

```diff
@@ -46,6 +47,9 @@ public class ModMetadata
 				var mod = FieldLoader.Load<ModMetadata>(nd["Metadata"]);
 				mod.Id = m;
 
+				if (nd.ContainsKey("ContentInstaller"))
+					mod.Content = FieldLoader.Load<ContentInstaller>(nd["ContentInstaller"]);
```

Comment:
The `ContentInstaller` block may exist, but have bogus content that makes FieldLoader throw.